src/entries/popup/pages/home/TabHeader.tsx

```diff
@@ -17,7 +17,7 @@ import { useUserAssetsBalance } from '../../hooks/useUserAssetsBalance';
 import { useVisibleTokenCount } from '../../hooks/useVisibleTokenCount';
 
 import DisplayModeDropdown from './NFTs/DisplayModeDropdown';
-import SortdDropdown from './NFTs/SortDropdown';
+import SortDropdown from './NFTs/SortDropdown';
```

Comment:
this typo is on the exported component name too, in the other file can't comment there